pkgs/games/terraria-server/linux.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, file, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name    = "terraria-server-linux-${version}";
+  version = "1308";
+
+  src = fetchurl {
+    url = http://terraria.org/server/terraria-server-linux-1308.tar.gz;
+    sha256 = "0cx3nx7wmzcw9l0nz9zm4amccl8nrd09hlb3jc1yrqcaswbyxc8a";
+  };
+
+  buildInputs = [ file ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r * $out/
+    ln -s $out/${name}/TerrariaServer.bin.x86_64 $out/bin/TerrariaServer
+
+    # Fix "/lib64/ld-linux-x86-64.so.2" like references in ELF executables.
+    echo "running patchelf on prebuilt binaries:"
+    find "$out" | while read filepath; do
+      if file "$filepath" | grep -q "ELF.*executable"; then
+        echo "setting interpreter $(cat "$NIX_CC"/nix-support/dynamic-linker) in $filepath"
+        patchelf --set-interpreter "$(cat "$NIX_CC"/nix-support/dynamic-linker)" "$filepath"
+        test $? -eq 0 || { echo "patchelf failed to process $filepath"; exit 1; }
+      fi
+    done
+  '';
+
+  meta = {
+    homepage = http://terraria.org;
+    description = "Dedicated server for the main game";
+    platforms = [ "x86_64-linux" ];
```

Comment:
What is the proper way to limit this package to x86_64-linux ?

platforms = platforms.x86_64; ?